### DISCUSSION TITLE
Multiple instances of MyEnergi integration?

### DISCUSSION THREAD

**Discussion by Johndowne** [CJNE/ha-myenergi#508] [Q&A] [answered]
Is it possible to have more than one MyEnergi instance for different buildings? If so what would be the simplest way to achieve that?
Many thanks for any insights.

John

**Comment by G6EJD** [comment, marked as answer]
Each building would need its own MyEnergi Hub / vHub with its own set of credentials. 
You can’t have multiple ME integrations running in HA.

**Reply by Johndowne** [reply]
Thanks for the clarification
